packages/faust-nx/src/config/index.ts

```diff
@@ -1,15 +1,29 @@
 import { WordPressTemplate } from '../getWordPressProps.js';
+import { hooks, Plugin } from '../hooks/index.js';
+import isEqual from 'lodash/isEqual';
 
 export interface FaustNXConfig {
   templates: { [key: string]: WordPressTemplate };
   disableLogging: boolean;
   loginPagePath?: string;
+  experimentalPlugins: Plugin[];
 }
 
-let config = {};
+let config: FaustNXConfig;
 
 export function setConfig(_config: FaustNXConfig) {
+  // Prevent plugins from being re-initialized.
+  if (isEqual(config, _config)) {
+    return;
+  }
+
   config = _config;
+
+  const { experimentalPlugins: plugins } = _config;
+
+  plugins?.forEach((plugin) => {
```

Comment:
Will the plugins get initiated only here whenever we call the setConfig? If yes we should make sure we call this setConfig only once. preferably have something like a global variable to track it or using
https://www.geeksforgeeks.org/lodash-_-once-method/ to prevent side effects. Maybe use a warning message as well in case the user calls this twice.